src/project/projectmodule.cpp

```diff
@@ -80,6 +82,7 @@ void ProjectModule::registerExports()
     ioc()->registerExport<IRecentFilesController>(moduleName(), m_recentFilesController);
     ioc()->registerExport<IOpenSaveProjectScenario>(moduleName(), new OpenSaveProjectScenario());
     ioc()->registerExport<IProjectFilesController>(moduleName(), m_actionsController);
+    ioc()->registerExport<IThumbnailCreator>(moduleName(), m_thumbnailCreator);
```

Comment:
just for information, if the object is not used in other methods (for example, we don't need to do init in onInit), then we don't have to create a class member (this is not a problem))
```
ioc()->registerExport<IThumbnailCreator>(moduleName(), new ThumbnailCreator());
```